src/components/Common/UserSelector.tsx

```diff
@@ -96,7 +116,18 @@ export default function UserSelector({
         align="start"
         sideOffset={4}
       >
-        <Command filter={() => 1}>
+        <Command
+          filter={(value, search) => {
+            const user = usersList.find((u) => u.id === value);
+            if (!user) return 0;
+
+            const name = formatName(user).toLowerCase();
+            const username = user.username?.toLowerCase() ?? "";
+            const input = search.toLowerCase();
+
+            return name.includes(input) || username.includes(input) ? 1 : 0;
+          }}
```

Comment:
let's not do this!

let's instead change the CommandItem's `value` to include both username and name instead!

let's stick with in-built behaviour of command for filtering please